accel/triton/ops_triton.py

```diff
@@ -109,21 +96,20 @@ def codegen(self):
     exec(codeObject, globals())
     program = globals()['fxn']
 
-    mem_estimate = sum(prod(x) for x in self.shapes)
     def runner(*bufs):
       GlobalCounters.global_ops += self.info.flops
-      GlobalCounters.global_mem += mem_estimate
+      GlobalCounters.global_mem += sum(prod(x.shape) for x in self.bufs)
```

Comment:
hmm, is this run every time the function is? now it's capturing self.bufs I believe

Reply:
I will fix it

Reply:
this is also wrong for a men estimate. see the one in GPU, if the shapetracker expands something that shouldn't be multiple mem accesses